demo.md

```diff
@@ -0,0 +1,155 @@
+# **RVIP** [Out The Box]
+**[Version]:** 1.0.0
+
+# **What Is This?**
+This document provides instruction to set-up **RVIP** as a complete Robotic Vision suite.
+
+This is based on the combination of ROS packages and external libraries used for testing purposes.
+
+This **suite** is made of:
+1. OpenGR
+2. darknet_ros
```

Comment:
`Installation link and instructions` have been provided within the same `demo.md` document. It can be found [here](https://github.com/cardboardcode/rvip-1/blob/master/demo.md#darknet_ros-setup).